{"name":"Course repository create course failure","fullName":"TestCourseCreateSuite/Course repository create course/TestCreate_Failure","status":"passed","statusDetails":{"message":"","trace":""},"start":1727875153107,"stop":1727875153107,"uuid":"eaad3c08-80c0-11ef-9e41-ca15b1947337","historyId":"925385c7eadee4db699a6f7a3ab970a8","testCaseId":"ceae5455df3ead54c52cceada24737b1","labels":[{"name":"language","value":"go1.22.5"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"macbook.local"},{"name":"thread","value":"TestCourseCreateSuite/Course repository create course/TestCreate_Failure"},{"name":"suite","value":"Course repository create course"},{"name":"package","value":"github.com/paw1a/eschool/internal/adapter/repository/postgres/test"}],"steps":[{"name":"ASSERT: Error Is","status":"passed","start":1727875153107,"stop":1727875153107,"parameters":[{"name":"Error","value":"sql: connection is already closed: persistence internal error"},{"name":"Target","value":"persistence internal error"}]}]}